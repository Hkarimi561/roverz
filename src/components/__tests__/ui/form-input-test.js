/**
 * Test to check if the component renders correctly
 */
/* global it expect */
import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import FormInput from '@ui/FormInput';
import FormInputSimple from '@ui/FormInputSimple';

it('FormInput renders correctly', () => {
  const tree = renderer.create(
    <FormInput value={'John Smith'} />,
  ).toJSON();

  expect(tree).toMatchSnapshot();
});


it('FormInputSimple renders correctly', () => {
  const tree = renderer.create(
    <FormInputSimple value={'John Smith'} />,
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
