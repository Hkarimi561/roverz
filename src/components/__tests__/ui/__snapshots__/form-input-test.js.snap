// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormInput renders correctly 1`] = `
<View
  style={
    Object {
      "borderBottomColor": "#bdc6cf",
      "borderBottomWidth": 1,
      "marginLeft": 20,
      "marginRight": 20,
      "transform": Array [
        Object {
          "translateX": 0,
        },
      ],
    }
  }
>
  <TextInput
    0={
      Object {
        "containerStyle": Array [
          Object {
            "backgroundColor": "rgba(255,255,255,0.05)",
            "borderBottomColor": "#D0D1D5",
            "borderBottomWidth": 1,
            "marginLeft": 20,
            "marginRight": 20,
            "marginTop": 10,
          },
        ],
        "inputStyle": Array [
          Object {
            "color": "#222222",
            "fontFamily": "OpenSans-Regular",
            "paddingHorizontal": 10,
            "paddingVertical": 3,
          },
        ],
        "value": "John Smith",
      }
    }
    1={Array []}
    2={Array []}
    selectionColor="#86939e"
    style={
      Array [
        Object {
          "color": "#86939e",
          "minHeight": 36,
          "width": 750,
        },
        Object {
          "fontSize": 17.5,
        },
        undefined,
      ]
    }
  />
</View>
`;

exports[`FormInputSimple renders correctly 1`] = `
<View
  style={
    Object {
      "borderBottomColor": "#bdc6cf",
      "borderBottomWidth": 1,
      "marginLeft": 20,
      "marginRight": 20,
      "transform": Array [
        Object {
          "translateX": 0,
        },
      ],
    }
  }
>
  <TextInput
    0={
      Object {
        "containerStyle": Array [
          Object {},
        ],
        "inputStyle": Array [
          Object {
            "color": "#777777",
            "fontFamily": "OpenSans-Regular",
            "paddingVertical": 3,
          },
        ],
        "value": "John Smith",
      }
    }
    1={Array []}
    2={Array []}
    selectionColor="#86939e"
    style={
      Array [
        Object {
          "color": "#86939e",
          "minHeight": 36,
          "width": 750,
        },
        Object {
          "fontSize": 17.5,
        },
        undefined,
      ]
    }
  />
</View>
`;
